test/pr_test.go

```diff
@@ -33,12 +33,14 @@ func TestRegistrationToCtxReleaseWithPDUSession(t *testing.T) {
 		Ip:   "127.0.0.1",
 		Port: 2154,
 	}
-	amfConfig := config.AMF{
-		Ip:   "127.0.0.1",
-		Port: 38414,
+	amfListConfig := []*config.AMF{
```

Comment:
Could you maybe add some tests with multiple AMFs in the next PR when you implement NG-FLEX?
Also, medium-term, I would like to add Free5GC to PacketRusher's CI, which might make more sense that implementing some dummy AMF/SMF/UPF/AUSF/UDM: TBD, as I would also like to turn PacketRusher into a kind of 5G Framework where you can instantiate some UE, gNodeB, some NFs all from Go code and make them interact together, (potentially with some NFs provided externally if the user chose so) similar to some commercial tools.

Reply:
I can't guarantee when I will write tests with multiple AMFs.